### PR TITLE
[irods/irods_2600] update irsync help text

### PR DESCRIPTION
**Pull request by trel**
just some words.

### CODE REVIEW

File: src/irsync.cpp [hunk 1]
```diff
@@ -169,53 +169,53 @@ usage() {
...
         "     cp -r foo1 foo2",
         " ",
-        "has a different meaning depending on whether the target directory foo2 ",
-        "exists. If the target directory exists, the content of source directory foo1",
-        "is copied to the target directory foo2/foo1. But if the target directory",
-        "does not exist, the content is copied to the target directory foo2.",
+        "has a different meaning depending on whether the target directory foo2",
+        "already exists. If the target directory exists, the content of source directory",
+        "foo1 is copied into the target directory foo2/foo1. But if the target directory",
+        "foo1 does not already exist, the content is copied as the target directory foo2.",
```
Comment (korydraughn) [inline src/irsync.cpp:193]: `foo1` is identified as source directory and target directory across these four lines.

Seems it should only be referred to as the source directory?

**Review by korydraughn — COMMENTED** [review]
Changes look good overall.

File: src/irsync.cpp (hunk 1 above)
Comment (trel) [reply src/irsync.cpp:193]: oh... i think there's a bug in the words.   looking again...

Comment (trel) [reply src/irsync.cpp]: yep, bug found - good eye.   updated in second commit.

File: src/irsync.cpp [hunk 2]
```diff
@@ -144,12 +144,12 @@ usage() {
         "          sourceFile|sourceDirectory [....] targetFile|targetDirectory",
         " ",
         "Synchronize the data between a local copy (local file system) and",
-        "the copy stored in iRODS or between two iRODS copies. The command can be ",
-        "in one of the three modes  : synchronization of data from the client's",
+        "the copy stored in iRODS or between two iRODS copies. The command can be",
+        "in one of the three modes : synchronization of data from the client's",
```
Comment (alanking) [inline src/irsync.cpp:148]:
```suggestion
        "in one of the three modes: synchronization of data from the client's",
```

**Review by alanking — COMMENTED** [review]
My suggestion is quite ignore-able

**Review by korydraughn — APPROVED** [review]
Looks good to me.

I think we're ready for the pound.

**Comment by trel** [comment]
#'d

**Pull request merged**